extensions/amp-autocomplete/validator-amp-autocomplete.protoascii

```diff
@@ -106,3 +106,14 @@ tags: {  # amp-autocomplete JSON
     }
   }
 }
+tags: {  # amp-autocomplete disabled
+  html_format: AMP
+  tag_name: "DIV"
```

Comment:
Are `div`s documented as the only possible element that can be `disabled` in amp-autocomplete?